lab_5_scraper/scraper.py

```diff
@@ -142,7 +330,52 @@ def get_search_urls(self) -> list:
         Returns:
             list: seed_urls param
         """
+        return self.config.get_seed_urls()
+
+class CrawlerRecursive(Crawler):
+    """
+    Recursive crawler implementation.
+    """
+
+    #: Url pattern
+    url_pattern: Union[Pattern, str]
+
+    def __init__(self, config: Config) -> None:
+        """
+        Initialize an instance of the CrawlerRecursive class.
+
+        Args:
+            config (Config): Configuration
+        """
+        super().__init__(config)
+        self.start_url = self.config.get_seed_urls()[0]
```

Comment:
So the crawler just picks one URL and works with it? Is there any way to use all seed_urls? E.g. if there are no links remaining from the first one, then we pick the second.